packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1296,92 +1393,23 @@ export class CrossChainMessenger {
       messageSlot
     )
 
-    // Sanity check that the value is set to 1 in the state
```

Comment:
Do you think this is a good sanity check? It would only break if we decided to start proving against a non bool in the state